release/sdk-config.yaml.tmpl

```diff
@@ -19,10 +19,22 @@ commands:
   path: damlc/damlc
   args: ["build", "--project-check"]
   desc: "Build the DAML project into a DAR file"
+
 - name: deploy
   path: daml-helper/daml-helper
-#  desc: "Build the DAML project and deploy to sandbox"
+  desc: "Build the DAML project and deploy to the ledger"
   args: ["deploy"]
+
+- name: list-parties
+  path: daml-helper/daml-helper
+  desc: "List parties known to the ledger"
+  args: ["list-parties"]
+
+- name: allocate-party
+  path: daml-helper/daml-helper
+  desc: "Allocate named party on the ledger"
+  args: ["allocate-party"]
+
```

Comment:
I don't think these should both be top level commands. Either we have:

* `daml party` with subcommands `daml party list` and `daml party allocate`, or
* these are all subcommands of `daml deploy`, as is the original `daml deploy` (could be `daml deploy package`, `daml deploy party`, `daml deploy party --list`, or so).

I'm partial to the latter (i.e. having all of these be subcommands of `daml deploy`), since the thing all of these commands have in common is that they need to communicate with a ledger on a given host/port.

Reply:
I'm open to the idea of structured commands. Is there any prior art for this in `daml` ?

Reply:
`da` used to have a bunch of them, and `daml install` sort of simulates this (with `daml install latest` / `daml install project` / `daml install VERSION`) but doesn't really do it the proper way, which is by using `subparser` and `command` in optparse-applicative.

Reply:
I  think we can leave this restructuring to a later PR. It will be nice to get more opinions form the produce people.